graphics/tinygl/api.cpp

```diff
@@ -283,6 +283,16 @@ void tglStencilOp(TGLenum sfail, TGLenum dpfail, TGLenum dppass) {
 	c->gl_add_op(p);
 }
 
+void tglPolygonStipple(const TGLubyte *mask) {
+	TinyGL::GLContext *c = TinyGL::gl_get_context();
+	TinyGL::GLParam p[2];
+
+	p[0].op = TinyGL::OP_PolygonStipple;
+	c->polygon_stipple_pattern = (void *)mask;
```

Comment:
I think this is wrong for 2 reasons: First, in line with previous comment, it should be safe to deallocate the mask memory as soon as this function returns, so the mask should be copied.  But, second, it looks like it is possible for `gl_add_op` to buffer the op, so the mask should be passed via `GLParam` similar to how `tglLoadMatrixf` works, and applied to the context when executing the op, not directly set on the context.

Reply:
I think this was fixed, please review.